hocon/src/main/scala/com/avsystem/commons/hocon/SizeInBytes.scala

```diff
@@ -0,0 +1,13 @@
+package com.avsystem.commons
+package hocon
+
+/**
+  * Use this type in data deserialized from HOCON to in order to read size in bytes represented with
+  * [[https://github.com/lightbend/config/blob/master/HOCON.md#size-in-bytes-format HOCON's nice representation]].
```

Comment:
Based on this source, there are a lot more strings supported in HOCON.

Reply:
My intention was to cover types supported by com.avsystem.commons.hocon.HoconTypeMarker

Reply:
Maybe just tune the doc a bit then to mention that